app/controllers/mortgage_calculator/land_and_buildings_transaction_taxes_controller.rb

```diff
@@ -0,0 +1,36 @@
+module MortgageCalculator
+  class LandAndBuildingsTransactionTaxesController < ::MortgageCalculator::ApplicationController
+    CALCULATOR = MortgageCalculator::LandAndBuildingsTransactionTax
+    before_action :set_rates
+    def show
+      @lbtt = CALCULATOR.new
+    end
+
+    def create
+      @lbtt = CALCULATOR.new(
+        params.require(:land_and_buildings_transaction_tax)
```

Comment:
@gotbadger I know this is used on a one off basis but shall this be made into a private method?

Reply:
don't think it adds much and its current format is how it appears in the stamp duty controller

Reply:
@gotbadger sure no np